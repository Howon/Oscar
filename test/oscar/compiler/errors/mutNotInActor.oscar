def main() => unit = {
  mut int x = 1; 
}
