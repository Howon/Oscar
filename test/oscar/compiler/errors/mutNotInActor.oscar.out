Error: Mutable types are only allowed in actors
